vm/src/stdlib/collections.rs

```diff
@@ -59,9 +59,9 @@ mod _collections {
         }
     }
 
-    struct SimpleSeqDeque<'a>(PyRwLockReadGuard<'a, VecDeque<PyObjectRef>>);
+    struct SimpleSeqDeque(VecDeque<PyObjectRef>);
```

Comment:
This is designed to borrow VecDeque but after this change, it will be cloned to make this struct.

Reply:
I switched to clone because it seemed like the easiest way to implement for the write guard as well. If you want I can switch to `
    struct SimpleSeqDeque<'a>(&'a VecDeque<PyObjectRef>);` -- or do you know if there is a trait that `PyRwLockReadGuard` and `PyRwLockWriteGuard` implement which I can use?
```suggestion
    struct SimpleSeqDeque<'a>(&'a VecDeque<PyObjectRef>);
```